packages/ui/Input/index.tsx

```diff
@@ -0,0 +1,168 @@
+import { useState, useRef, useId } from "react";
+import styled, { css } from "styled-components";
+
+import { Icon } from "../Icon";
+
+type StyledErrorProps = {
+  hasError: boolean;
+};
+
+export type InputProps = {
+  label: string;
+  supportingLabel?: React.ReactNode;
+  onChange?: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  onInputCleared?: () => void;
+} & StyledErrorProps &
+  React.HTMLProps<HTMLInputElement>;
```

Comment:
@monika-klokosinska-intive @wlechowicz should I limit the props from input element only to those that are used by the component? https://www.typescriptlang.org/docs/handbook/utility-types.html#picktype-keys

it could look like this:
```suggestion
  Pick<React.HTMLProps<HTMLInputElement>, 'id' | 'type' | 'value'>;
```

Reply:
@wyola either way is fine with me! @wlechowicz what do you think?